### PR TITLE
Add a simple image uploader partial

### PR DESCRIPTION
**Pull request by jacobherrington**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Adds a little image uploading widget to the admin area. This partial can be rendered on any page, so it might be neat to think of a pattern that makes this tool (and others like it) more generally available kind of like how the mod tools work on article pages.

I spent some time thinking about improvements that needed to be made to image management on the site and started working on those, but scrapped that in order to focus on solving the immediate need.

## Related Tickets & Documents

Closes https://github.com/forem/InternalProjectPlanning/issues/122

## QA Instructions, Screenshots, Recordings

image

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

**Review by juliannatetreault — COMMENTED** [review]
From what I understand, the goal of this is to just create an image uploading tool that Admins can render/use anywhere that they need/want to quickly upload an image, correct @jacobherrington? When I tested this out locally it worked great but I want to make sure that I understand what the need for this is. 😄

**Comment by jacobherrington** [comment]
@juliannatetreault That's my understanding! There used to be one of these in another part of the admin interface, but it was removed about a year ago.

File: app/javascript/admin/controllers/image_upload_controller.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { Controller } from 'stimulus';
+
+export default class ImageUploadController extends Controller {
```
Comment (rhymes) [inline app/javascript/admin/controllers/image_upload_controller.js:3]: ❤️ ❤️ ❤️

**Review by rhymes — APPROVED** [review]
Tested it, works great!

**Review by citizen428 — APPROVED** [review]

**Review by juliannatetreault — APPROVED** [review]
This is such a nice tool for Admins and works great! ✨

**Comment by leewynne** [comment]
This is great!

**Pull request merged**